src/world/mod.rs

```diff
@@ -34,14 +34,16 @@ impl<'a> Iterator for CreateIter<'a> {
     }
 }
 
-/// A common trait for EntityBuilder and LazyBuilder, allowing either to be used.
+/// A common trait for [`EntityBuilder`] and [`LazyBuilder`], allowing either to be used.
+/// Entity is definitely alive, but the components may or may not exist before a call to
+/// [`World::maintain`].
 pub trait Builder {
     /// Appends a component and associates it with the entity.
     ///
     /// # Panics
     ///
-    /// Panics if the component hasn't been `register()`ed in the
-    /// `World`.
+    /// Panics if the component hasn't been [`register()`](World::register)ed in the
```

Comment:
Do these links really work? I never saw them like this.

Reply:
Hmm. They work for nightly, but not stable apparently. The ``[`Thing`]`` ones without specifying the link later don't work without nightly either,. The re-export ones don't work (as you were saying) without `#[doc(no-inline)]`, but using that puts it in the re-exports section with no description...

If you would prefer, I'll just take out all the links.